uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java

```diff
@@ -220,18 +187,23 @@ public void testCreateDataFormat() {
 	 */
 	@Test
 	public void testEditDataFormat() {
+		createDataFormat();
+		String[] path = { ROOT, dataFormatName + " (Data Format)" };
+		WaitCondition wait = new TreeHasItem(new DefaultTree(editor), path);
+		new WaitUntil(wait, TimePeriod.MEDIUM, false);
+		if (wait.getResult() != null) {
+			assertTrue("Tree does not contains DataFormat - " + dataFormatName, wait.getResult());
+		}
+
+		editor.editDataFormat(dataFormatName);
+		new LabeledText("Id").setText("new_" + dataFormatName);
```

Comment:
I assume that the previous line opens a dialog/window. So, it would be great if the method editDataFormat return the dialog. Then, we could use something like this
```
DataFormatEditDialog editDialog =  editor.editDataFormat(dataFormatName);
editDialog.setId(("new_" + dataFormatName);
```
or at least do the following
```
editor.editDataFormat(dataFormatName);
Shell editDialog = new DefaultShell("The edit dialog title");
new LabeledText(editDialog, "Id").setText("new_" + dataFormatName);
```